tests/e2e/e2e_fluentbit_test.go

```diff
@@ -0,0 +1,61 @@
+package e2e
```

Comment:
@shashankram  @nojnhuh , we need to cut down the CI and gating tests, we should meet to draw the line. 
Additionally we have some additional concerns regarding parallelization, gh actions might run oversubscribed with multiple kind clusters.

I'll schedule a meeting to agree on path forward, threshold of acceptable CI window time, etc.